archivers/xz/Portfile

```diff
@@ -22,7 +25,6 @@ XZ utils consist of a few relatively separate submodules: \
 \n* Scripts to ease grepping, diffing and viewing (lz*grep, lzdiff/lzcmp, lzmore/lzless)
 
 homepage        https://tukaani.org/xz/
```

Comment:
```suggestion
homepage        https://tukaani.org/xz/
github.tarball_from \
                releases
```

Reply:
Hi @ryandesign, I have made these changes to the Portfile and have updated my commit. Thanks.